### PR TITLE
cors config on route POST /forms

### PR DESCRIPTION
**Pull request by Youllou**
Not ready yet !

### CODE REVIEW

**Review by k-allagbe — APPROVED** [review]
Tested, working as expected.

File: app.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def verify_password(user_id, password):
 
 @app.route('/forms', methods=['POST'])
 @auth.login_required
-@cross_origin(origins=FRONTEND_URL)
+@cross_origin(origins='*')
```
Comment (k-allagbe) [inline app.py:67]: Why?

Comment (snakedye) [reply app.py:67]: The test fails otherwise. I'll try to put an array with the front-end URL and localhost.

Comment (k-allagbe) [reply app.py]: Strange because manual tests with local instances worked.

**Pull request merged**